stacks_queues/stack.py

```diff
@@ -21,18 +22,22 @@ def pop(self):
             The Stack is empty.
             returns None
         """
-        pass
+        if self.empty():
+            raise StackEmptyException("Stack is empty")
+            return None
+        else:
+            return self.store.remove_first()
 
     def empty(self):
         """ Returns True if the Stack is empty
             And False otherwise
         """
-        pass
+        return self.store.length() == 0
 
     def __str__(self):
         """ Returns the Stack in String form like:
             [3, 4, 7]
             Starting with the top of the Stack and
             ending with the bottom of the Stack.
         """
-        pass
+        return self.store.__str__()
```

Comment:
Dunder methods (methods with the double underscore on either side) are called without the dunders, so here `__str__` becomes `str`. The double underscores indicate we are doing something called operator overloading which basically means we redefine or extend what we want a built-in function like `str` to do to our class. 
```suggestion
        return str(self.store)
```